python/ray/_private/worker.py

```diff
@@ -1862,6 +1863,7 @@ def is_initialized() -> bool:
     return ray._private.worker.global_worker.connected
 
 
+@metrics.monitor("worker.connect")
```

Comment:
nit: would `_` be a better delimiter? I assume it will eventuall expand to `ray_worker_connect_total` rather than `ray_worker.connect.total`?